tensorflow/core/kernels/conv_ops_fused_int8.cc

```diff
@@ -261,7 +261,7 @@ struct LaunchFusedConv2DOpCpuInt8Helper {
         }
 
         // (2) Side input.
-        if (side_input_scale != 0.0f) {
+        if (side_input_scale != 0.0f && side_input_ptr != nullptr) {
```

Comment:
`side_input_ptr` undefined at this line

Reply:
My bad, I corrected it to refer `side_input_base` instead of `side_input_ptr`